PullRequestCreated/PullRequestCreated.ts

```diff
@@ -16,6 +22,25 @@ class PullRequestCreated extends OctokitAction {
   private shouldCreateIssue(pr: PullRequest): boolean {
     return pr != null && !JIRA_ISSUE_PATTERN.test(pr.title);
   }
+
+  private async newIssueTypeAndParent(pr: PullRequest): Promise<Issue> {
+    const mentionnedIssues = this.findMentionnedIssues(pr);
+    console.log(`Found ${mentionnedIssues.length} mentionned issues: ${mentionnedIssues}`);
+    let parent = mentionnedIssues.length === 1 ? await this.jira.getIssue(mentionnedIssues[0]) : null;
+    console.log(`Parent issue: ${parent?.key} (${parent?.fields.issuetype.name})`);
+    switch (parent?.fields.issuetype.name) {
+      case 'Epic':
+        return { type: 'Task', parent: parent.key };
+      case 'Task':
+        return { type: 'Sub-task', parent: parent.key };
```

Comment:
This is wrong, it's not `Task`, it's any issue except subtasks and epics. Imagine `NewFeature`, `Improvement`, `False-Positive`, `False-Negative`...